hBank/src/main/java/animalPark/KingdomApp.java

```diff
@@ -0,0 +1,36 @@
+package animalPark;
+
+import animalPark.animals.Cat;
+import animalPark.animals.Dog;
+import animalPark.animals.Lion;
+import animalPark.animals.Wolf;
+
+public class KingdomApp {
+    public static void main(String[] args) {
```

Comment:
Create a instance method that generates 5 (of your choice) objects and returns list of them as List<AnimalInterface>
additionally add a test that validates this method

Reply:
Done